account/identity.go

```diff
@@ -138,6 +142,28 @@ func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Ident
 	return &native, errs.WithStack(err)
 }
 
+// Exists returns true|false where an object exists with an identifier
+func (m *GormIdentityRepository) Exists(ctx context.Context, id uuid.UUID) (bool, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "identity", "exists"}, time.Now())
+
+	queryStmt, err := m.db.CommonDB().Prepare(fmt.Sprintf(`
```

Comment:
@hectorj2f In 6a1155b01acb58a99d082fa2adf34b6aba1c7829 you're no longer using prepared statements, are you?